.github/workflows/ci.yml

```diff
@@ -28,7 +28,9 @@ jobs:
           sudo apt-get update && \
           sudo apt-get install -yq libasound2-dev libsoxr-dev libvorbisidec-dev libvorbis-dev libflac-dev \
           libopus-dev alsa-utils libpulse-dev libavahi-client-dev libssl-dev \
-          avahi-daemon ccache expat cppcheck clang-tidy
+          avahi-daemon ccache expat cppcheck clang-tidy \
+          libpipewire-0.3-dev pipewire-pulse libdbus-1-dev libjack-dev libavahi-client-dev \
```

Comment:
I think these are not needed and/or duplicates `ibdbus-1-dev libjack-dev libavahi-client-dev libatomic1  libvorbis-dev libopus-dev libpulse-dev libflac-dev libsoxr-dev libexpat1-dev`